node-via-nvm/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+NODE_VERSIONS=("${NODE_VERSIONS}")
```

Comment:
Parenthesis give the wrong impression that this will be an array. In theory, the `"` could also lead to variable expansion given the right kind of input, so let's change that to `'` instead of `"`, we're not dealing with a shell variable anyway.

```suggestion
NODE_VERSIONS='${NODE_VERSIONS}'
```